test/package/test_package_fx.py

```diff
@@ -17,19 +17,31 @@
     from common import PackageTestCase
 
 
+class CustomTracer(Tracer):
+    def is_leaf_module(self, m, module_qualified_name,) -> bool:
+        return super().is_leaf_module(m, module_qualified_name) or module_qualified_name == "MyModule"
```

Comment:
Is there anything else that's simple yet unique that I can do in this tracer to make it super obvious when a `Graph` is produced using it? I think this does basically the same thing as the `KeepModules` tracer from `GraphModule` deserialization, so I don't think the graphs produced by the two would differ.